### PR TITLE
Use Python 'int' and 'float' type to avoid numpy warnings

### PR DESCRIPTION
**Pull request by mcara**
This replaces the deprecated Numpy globals

```python
np.float
np.int
```

with the Python types

```python
float
int
```
 
to avoid (reduce) deprecation warnings.

### CODE REVIEW

File: lib/stsci/tools/nmpfit.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 """
-Python/Numeric version of this module was called mpfit. This version was modified to use numpy.
+Python/Numeric version of this module was called mpfit. This version was modified to use np.
```
Comment (pllim) [inline lib/stsci/tools/nmpfit.py:2]: Your `sed` got a little too aggressive? 
```suggestion
Python/Numeric version of this module was called mpfit. This version was modified to use numpy.
```

**Review by pllim — APPROVED** [review]
`gfit` and `nmpfit` are both dead (see #108) but given they can't really die because of reasons, might as well, eh?

Looks good with minor comments below.

**Review by nden — APPROVED** [review]

**Comment by nden** [comment]
I'm starting to think we should move nmpfit and gfit to stistools, the only known use case.

**Comment by mcara** [comment]
> I'm starting to think we should move nmpfit and gfit to stistools, the only known use case.

Please not in this PR 🤞

**Comment by pllim** [comment]
I don't even know who manages stistools nowadays. Probably INS.

**Pull request merged**